tests/core/remotable_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Please put all of the test files into tests/

Reply:
what do you mean? this is in `tests/core/` already?